To update maximizer connector

davidchanmaximizer opened the pull request:
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

davidchanmaximizer committed:
update authenticate method

davidchanmaximizer committed:
Update icon brand color

davidchanmaximizer committed:
updated response schema

davidchanmaximizer committed:
Merge branch 'dev' of https://github.com/davidchanmaximizer/PowerPlatformConnectors into dev

davidchanmaximizer committed:
Merge branch 'microsoft:dev' into dev

davidchanmaximizer committed:
Merge branch 'microsoft:dev' into dev

davidchanmaximizer committed:
update response schema

davidchanmaximizer committed:
Merge branch 'dev' of https://github.com/davidchanmaximizer/PowerPlatformConnectors into dev

davidchanmaximizer committed:
Merge branch 'dev' of https://github.com/davidchanmaximizer/PowerPlatformConnectors into dev

davidchanmaximizer committed:
updated swagger

vmanoharas reviewed (commented):
Hello @davidchanmaximizer, 

The swagger file is very large I cannot view to comment on which line the swagger validation error is. Kindly review below swagger validation errors and fix them. 


##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerNewLead Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerUpdateLead Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsTextbox Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsCreate Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsUpdate Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/FindAbEntry Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownAbEntryUdf Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryTextbox Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABAddrTxtbox Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryCreate Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryUpdate Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryAddressUpdate Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownCity Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DpdnStPv Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TasksTextbox Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TskHtlCrt Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppTxtBox Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppCreate Source: NewSwagger

davidchanmaximizer commented:
Hi vmanoharas,

it is because we renamed most of the action, in this case, may I know what we can do?

Best Regards,
David CHAN
________________________________
From: vmanoharas ***@***.***>
Sent: Friday, May 10, 2024 01:34
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: David Chan ***@***.***>; Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470)


External Message: Please be cautious when opening links or attachments in email.

@vmanoharas commented on this pull request.

Hello @davidchanmaximizer<https://github.com/davidchanmaximizer>,

The swagger file is very large I cannot view to comment on which line the swagger validation error is. Kindly review below swagger validation errors and fix them.

##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerNewLead Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerUpdateLead Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsTextbox Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsCreate Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsUpdate Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/FindAbEntry Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownAbEntryUdf Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryTextbox Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABAddrTxtbox Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryCreate Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryUpdate Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryAddressUpdate Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownCity Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DpdnStPv Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TasksTextbox Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TskHtlCrt Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppTxtBox Source: NewSwagger
##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppCreate Source: NewSwagger

—
Reply to this email directly, view it on GitHub<https://github.com/microsoft/PowerPlatformConnectors/pull/3470#pullrequestreview-2048476706>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUA6SYAI27KBZXHZUXMA2XTZBOXTLAVCNFSM6AAAAABHOH64LKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDANBYGQ3TMNZQGY>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

vmanoharas commented:
> Hi vmanoharas, it is because we renamed most of the action, in this case, may I know what we can do? Best Regards, David CHAN
> […](#)
> ________________________________ From: vmanoharas ***@***.***> Sent: Friday, May 10, 2024 01:34 To: microsoft/PowerPlatformConnectors ***@***.***> Cc: David Chan ***@***.***>; Mention ***@***.***> Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470) External Message: Please be cautious when opening links or attachments in email. @vmanoharas commented on this pull request. Hello @davidchanmaximizer<https://github.com/davidchanmaximizer>, The swagger file is very large I cannot view to comment on which line the swagger validation error is. Kindly review below swagger validation errors and fix them. ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerNewLead Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerUpdateLead Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsCreate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/FindAbEntry Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownAbEntryUdf Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABAddrTxtbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryCreate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryAddressUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownCity Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DpdnStPv Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TasksTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TskHtlCrt Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppTxtBox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppCreate Source: NewSwagger — Reply to this email directly, view it on GitHub<[#3470 (review)](https://github.com/microsoft/PowerPlatformConnectors/pull/3470#pullrequestreview-2048476706)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUA6SYAI27KBZXHZUXMA2XTZBOXTLAVCNFSM6AAAAABHOH64LKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDANBYGQ3TMNZQGY>. You are receiving this because you were mentioned.Message ID: ***@***.***>

Hello @davidchanmaximizer, 

All are breaking changes, your existing flows will fail after certification, this same question will be asking during certification as well.  Kindly confirm that your existing flows will not be failed. so that I can approve. Additionally kindly agree CLA Agreement.

davidchanmaximizer commented:
Hi Vamanoharas,

As you said it is a breaking change, I think all existing flows will be failed, because we updated all response schema.
And I remember we need to inform all existing user, may I know how can I do it?

Best Regards,
David CHAN
________________________________
From: vmanoharas ***@***.***>
Sent: Friday, May 10, 2024 16:06
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: David Chan ***@***.***>; Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470)


External Message: Please be cautious when opening links or attachments in email.

Hi vmanoharas, it is because we renamed most of the action, in this case, may I know what we can do? Best Regards, David CHAN
…
________________________________ From: vmanoharas @.> Sent: Friday, May 10, 2024 01:34 To: microsoft/PowerPlatformConnectors @.> Cc: David Chan @.>; Mention @.> Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470<https://github.com/microsoft/PowerPlatformConnectors/pull/3470>) External Message: Please be cautious when opening links or attachments in email. @vmanoharas<https://github.com/vmanoharas> commented on this pull request. Hello @davidchanmaximizer<https://github.com/davidchanmaximizer>https://github.com/davidchanmaximizer, The swagger file is very large I cannot view to comment on which line the swagger validation error is. Kindly review below swagger validation errors and fix them. ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerNewLead Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerUpdateLead Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsCreate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/FindAbEntry Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownAbEntryUdf Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABAddrTxtbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryCreate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryAddressUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownCity Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DpdnStPv Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TasksTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TskHtlCrt Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppTxtBox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppCreate Source: NewSwagger — Reply to this email directly, view it on GitHub<#3470 (review)<https://github.com/microsoft/PowerPlatformConnectors/pull/3470#pullrequestreview-2048476706>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/AUA6SYAI27KBZXHZUXMA2XTZBOXTLAVCNFSM6AAAAABHOH64LKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDANBYGQ3TMNZQGY. You are receiving this because you were mentioned.Message ID: @.***>

Hello @davidchanmaximizer<https://github.com/davidchanmaximizer>,

All are breaking changes, your existing flows will fail after certification, this same question will be asking during certification as well. Kindly confirm that your existing flows will not be failed. so that I can approve.

—
Reply to this email directly, view it on GitHub<https://github.com/microsoft/PowerPlatformConnectors/pull/3470#issuecomment-2104141742>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUA6SYHKVPRFJSX4IWYU6FTZBR5YLAVCNFSM6AAAAABHOH64LKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMBUGE2DCNZUGI>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

davidchanmaximizer commented:
Hi Vamanoharas,

May I know if we can apply a new custom connector for Maximer Connector 2.0?
It is a workaround of it?

Best Regards,
David CHAN
________________________________
From: David Chan ***@***.***>
Sent: Monday, May 13, 2024 09:38
To: microsoft/PowerPlatformConnectors ***@***.***>; microsoft/PowerPlatformConnectors ***@***.***>
Cc: Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470)

Hi Vamanoharas,

As you said it is a breaking change, I think all existing flows will be failed, because we updated all response schema.
And I remember we need to inform all existing user, may I know how can I do it?

Best Regards,
David CHAN
________________________________
From: vmanoharas ***@***.***>
Sent: Friday, May 10, 2024 16:06
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: David Chan ***@***.***>; Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470)


External Message: Please be cautious when opening links or attachments in email.

Hi vmanoharas, it is because we renamed most of the action, in this case, may I know what we can do? Best Regards, David CHAN
…
________________________________ From: vmanoharas @.> Sent: Friday, May 10, 2024 01:34 To: microsoft/PowerPlatformConnectors @.> Cc: David Chan @.>; Mention @.> Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470<https://github.com/microsoft/PowerPlatformConnectors/pull/3470>) External Message: Please be cautious when opening links or attachments in email. @vmanoharas<https://github.com/vmanoharas> commented on this pull request. Hello @davidchanmaximizer<https://github.com/davidchanmaximizer>https://github.com/davidchanmaximizer, The swagger file is very large I cannot view to comment on which line the swagger validation error is. Kindly review below swagger validation errors and fix them. ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerNewLead Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerUpdateLead Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsCreate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/FindAbEntry Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownAbEntryUdf Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABAddrTxtbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryCreate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryAddressUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownCity Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DpdnStPv Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TasksTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TskHtlCrt Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppTxtBox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppCreate Source: NewSwagger — Reply to this email directly, view it on GitHub<#3470 (review)<https://github.com/microsoft/PowerPlatformConnectors/pull/3470#pullrequestreview-2048476706>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/AUA6SYAI27KBZXHZUXMA2XTZBOXTLAVCNFSM6AAAAABHOH64LKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDANBYGQ3TMNZQGY. You are receiving this because you were mentioned.Message ID: @.***>

Hello @davidchanmaximizer<https://github.com/davidchanmaximizer>,

All are breaking changes, your existing flows will fail after certification, this same question will be asking during certification as well. Kindly confirm that your existing flows will not be failed. so that I can approve.

—
Reply to this email directly, view it on GitHub<https://github.com/microsoft/PowerPlatformConnectors/pull/3470#issuecomment-2104141742>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUA6SYHKVPRFJSX4IWYU6FTZBR5YLAVCNFSM6AAAAABHOH64LKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMBUGE2DCNZUGI>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

davidchanmaximizer commented:
Hi Vamanohars,

Also, may I know how many users using our custom connector?

Best Regards,
David CHAN
________________________________
From: David Chan ***@***.***>
Sent: Tuesday, May 14, 2024 08:10
To: microsoft/PowerPlatformConnectors ***@***.***>; microsoft/PowerPlatformConnectors ***@***.***>
Cc: Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470)

Hi Vamanoharas,

May I know if we can apply a new custom connector for Maximer Connector 2.0?
It is a workaround of it?

Best Regards,
David CHAN
________________________________
From: David Chan ***@***.***>
Sent: Monday, May 13, 2024 09:38
To: microsoft/PowerPlatformConnectors ***@***.***>; microsoft/PowerPlatformConnectors ***@***.***>
Cc: Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470)

Hi Vamanoharas,

As you said it is a breaking change, I think all existing flows will be failed, because we updated all response schema.
And I remember we need to inform all existing user, may I know how can I do it?

Best Regards,
David CHAN
________________________________
From: vmanoharas ***@***.***>
Sent: Friday, May 10, 2024 16:06
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: David Chan ***@***.***>; Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470)


External Message: Please be cautious when opening links or attachments in email.

Hi vmanoharas, it is because we renamed most of the action, in this case, may I know what we can do? Best Regards, David CHAN
…
________________________________ From: vmanoharas @.> Sent: Friday, May 10, 2024 01:34 To: microsoft/PowerPlatformConnectors @.> Cc: David Chan @.>; Mention @.> Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470<https://github.com/microsoft/PowerPlatformConnectors/pull/3470>) External Message: Please be cautious when opening links or attachments in email. @vmanoharas<https://github.com/vmanoharas> commented on this pull request. Hello @davidchanmaximizer<https://github.com/davidchanmaximizer>https://github.com/davidchanmaximizer, The swagger file is very large I cannot view to comment on which line the swagger validation error is. Kindly review below swagger validation errors and fix them. ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerNewLead Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerUpdateLead Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsCreate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/FindAbEntry Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownAbEntryUdf Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABAddrTxtbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryCreate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryAddressUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownCity Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DpdnStPv Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TasksTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TskHtlCrt Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppTxtBox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppCreate Source: NewSwagger — Reply to this email directly, view it on GitHub<#3470 (review)<https://github.com/microsoft/PowerPlatformConnectors/pull/3470#pullrequestreview-2048476706>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/AUA6SYAI27KBZXHZUXMA2XTZBOXTLAVCNFSM6AAAAABHOH64LKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDANBYGQ3TMNZQGY. You are receiving this because you were mentioned.Message ID: @.***>

Hello @davidchanmaximizer<https://github.com/davidchanmaximizer>,

All are breaking changes, your existing flows will fail after certification, this same question will be asking during certification as well. Kindly confirm that your existing flows will not be failed. so that I can approve.

—
Reply to this email directly, view it on GitHub<https://github.com/microsoft/PowerPlatformConnectors/pull/3470#issuecomment-2104141742>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUA6SYHKVPRFJSX4IWYU6FTZBR5YLAVCNFSM6AAAAABHOH64LKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMBUGE2DCNZUGI>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

vmanoharas commented:
Hello @davidchanmaximizer, 

Unfortunately, we will not be maintaining users list with us.  I would suggest you to deprecate this connector and submit a new connector. Kindly follow the instructions in the document to deprecate the connector. https://learn.microsoft.com/en-us/connectors/custom-connectors/deprecate-operation

vmanoharas commented:
> Hi Vamanoharas, May I know if we can apply a new custom connector for Maximer Connector 2.0? It is a workaround of it? Best Regards, David CHAN
> […](#)
> ________________________________ From: David Chan ***@***.***> Sent: Monday, May 13, 2024 09:38 To: microsoft/PowerPlatformConnectors ***@***.***>; microsoft/PowerPlatformConnectors ***@***.***> Cc: Mention ***@***.***> Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470) Hi Vamanoharas, As you said it is a breaking change, I think all existing flows will be failed, because we updated all response schema. And I remember we need to inform all existing user, may I know how can I do it? Best Regards, David CHAN
> ________________________________ From: vmanoharas ***@***.***> Sent: Friday, May 10, 2024 16:06 To: microsoft/PowerPlatformConnectors ***@***.***> Cc: David Chan ***@***.***>; Mention ***@***.***> Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470) External Message: Please be cautious when opening links or attachments in email. Hi vmanoharas, it is because we renamed most of the action, in this case, may I know what we can do? Best Regards, David CHAN …
> ________________________________ From: vmanoharas @.> Sent: Friday, May 10, 2024 01:34 To: microsoft/PowerPlatformConnectors @.> Cc: David Chan @.>; Mention @.> Subject: Re: [microsoft/PowerPlatformConnectors] To update maximizer connector (PR #3470<#3470>) External Message: Please be cautious when opening links or attachments in email. @vmanoharas<https://github.com/vmanoharas> commented on this pull request. Hello @davidchanmaximizer<https://github.com/davidchanmaximizer>https://github.com/davidchanmaximizer, The swagger file is very large I cannot view to comment on which line the swagger validation error is. Kindly review below swagger validation errors and fix them. ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerNewLead Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TriggerUpdateLead Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsCreate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/LeadsUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/FindAbEntry Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownAbEntryUdf Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABAddrTxtbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryCreate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/ABEntryAddressUpdate Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DropdownCity Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/DpdnStPv Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TasksTextbox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/TskHtlCrt Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppTxtBox Source: NewSwagger ##[error]The operation has been deleted. This change breaks existing flows. ValuePath: operation/AppCreate Source: NewSwagger — Reply to this email directly, view it on GitHub<#3470 (review)<[#3470 (review)](https://github.com/microsoft/PowerPlatformConnectors/pull/3470#pullrequestreview-2048476706)>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/AUA6SYAI27KBZXHZUXMA2XTZBOXTLAVCNFSM6AAAAABHOH64LKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDANBYGQ3TMNZQGY. You are receiving this because you were mentioned.Message ID: @.***> Hello @davidchanmaximizer<https://github.com/davidchanmaximizer>, All are breaking changes, your existing flows will fail after certification, this same question will be asking during certification as well. Kindly confirm that your existing flows will not be failed. so that I can approve. — Reply to this email directly, view it on GitHub<[#3470 (comment)](https://github.com/microsoft/PowerPlatformConnectors/pull/3470#issuecomment-2104141742)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUA6SYHKVPRFJSX4IWYU6FTZBR5YLAVCNFSM6AAAAABHOH64LKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMBUGE2DCNZUGI>. You are receiving this because you were mentioned.Message ID: ***@***.***>

Yes @davidchanmaximizer, as I commented just now, please deprecate this connector and submit a new connector. Please follow the deprecate connector document below and submit this connector. 
https://learn.microsoft.com/en-us/connectors/custom-connectors/deprecate-operation

davidchanmaximizer committed:
Merge branch 'microsoft:dev' into dev
